quickwit-cli/src/main.rs

```diff
@@ -116,6 +117,34 @@ async fn jemalloc_metrics_loop() -> tikv_jemalloc_ctl::Result<()> {
     }
 }
 
+/// If a bunch of tokio runtimes need to be started for actors,
+/// return the right configuration.
+///
+/// TODO making it configurable could be useful in the
```

Comment:
`/// TODO making it configurable could be useful in the` future?

Reply:
ok repetition at the end, so you can ditch "in the".

Reply:
fixed